### PR TITLE
[PTI-553][PTI-552] Fixed several issues with advanced filters. Set subset cron to 5 min …

### PR DESCRIPTION
**Pull request by danieltruong**
…intervals

### CODE REVIEW

**Comment by NickPhura** [comment]
Admin
- Clicking "Reset Filters" doesn't clear the Entity Type checkboxes.

Nrced
- Something is wrong with the sortBy.  It looks like the request only ever sends `sortBy=-dateAdded` regardless of what the sortBy picklist is set to.

**Comment by NickPhura** [comment]
Admin/Nrced
- Possibly a side effect of the above sortBy issue: if you change the sorting and copy the URL.  When you load that URL, the UI for the columnHeader/sortBy picklist doesn't reflect the soryBy url param.

File: angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts [hunk 1]
```diff
@@ -379,25 +295,28 @@ export class RecordsListComponent implements OnInit, OnDestroy {
    * @memberof SearchComponent
    */
   submit() {
+    this.queryParams['ms'] = new Date().getMilliseconds();
     this.loadingScreenService.setLoadingState(true, 'body');
-    this.tableTemplateUtils.navigateUsingParams(this.tableData, ['records']);
+    console.log({ ...this.queryParams, ...this.tableTemplateUtils.getNavParamsObj(this.tableData) });
```
Comment (NickPhura) [inline angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts:300]: nit: console log

**Comment by danieltruong** [comment]
> Admin/Nrced
> 
> * Possibly a side effect of the above sortBy issue: if you change the sorting and copy the URL.  When you load that URL, the UI for the columnHeader/sortBy picklist doesn't reflect the soryBy url param.

Did you run build global?

**Comment by NickPhura** [comment]
> 
> 
> > Admin/Nrced
> > 
> > * Possibly a side effect of the above sortBy issue: if you change the sorting and copy the URL.  When you load that URL, the UI for the columnHeader/sortBy picklist doesn't reflect the soryBy url param.
> 
> Did you run build global?

I believe I did, but ill re-build it all just in case

**Comment by NickPhura** [comment]
Did a new build and I still get the same issues I reported above

File: angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts (hunk 1 above)
Comment (danieltruong) [reply angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts]: 5dbb927f2fd0afe350d29d161094e005ed715dec

**Comment by danieltruong** [comment]
> Admin
> 
> * Clicking "Reset Filters" doesn't clear the Entity Type checkboxes.
> 
> Admin/Nrced
> 
> * Something is wrong with the sortBy.  It looks like the request only ever sends `sortBy=-dateAdded` regardless of what the columnHeader/sortBy picklist is set to or whats in the url.

Reset issue fixed in 5dbb927f2fd0afe350d29d161094e005ed715dec

**Comment by danieltruong** [comment]
> > > Admin/Nrced
> > > 
> > > * Possibly a side effect of the above sortBy issue: if you change the sorting and copy the URL.  When you load that URL, the UI for the columnHeader/sortBy picklist doesn't reflect the soryBy url param.
> > 
> > 
> > Did you run build global?
> 
> I believe I did, but ill re-build it all just in case

Found the issue. Fixed in f8a6547115e53019f601e7bbab948f396e897bae

**Comment by NickPhura** [comment]
nice one, looks good!

**Pull request merged**